### PR TITLE
TEP-0114: Enable Custom Tasks in Pipelines by Default

### PR DESCRIPTION
**Pull request by jerop**
# Changes

In this change, we enable Custom Tasks in Pipelines by default by flipping `enable-custom-tasks` from `"false"` to `"true"`.

We will remove this feature flag soon.

See issue - https://github.com/tektoncd/pipeline/issues/5155.

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [x] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [x] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [x] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
Custom Tasks in Pipelines are enabled by default. To disable, set `enable-custom-tasks` feature flag `"false"`.
```

### CODE REVIEW

**Comment by jerop** [comment]
/hold 

wait for https://github.com/tektoncd/pipeline/pull/5832 to merge

**Comment by jerop** [comment]
cc @abayer @XinruZhang

**Review by chitrangpatel — COMMENTED** [review]
I wonder if https://github.com/tektoncd/pipeline/blob/3279a4310399655b78df0bfeb805a931b0e84481/test/custom_task_test.go#L59 needs to be updated as well.

**Review by chitrangpatel — APPROVED** [review]

**Comment by jerop** [comment]
https://github.com/tektoncd/pipeline/pull/5832 is merged

/hold cancel

**Comment by jerop** [comment]
@tektoncd/core-maintainers please take a look 🙏🏾

**Review by XinruZhang — APPROVED** [review]

**Pull request merged**
